tests/by-util/test_cksum.rs

```diff
@@ -435,6 +443,23 @@ fn test_all_algorithms_fail_on_folder() {
     }
 }
 
+#[test]
+fn test_binary_file() {
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+
+    at.touch("f");
+
+    scene
+        .ucmd()
+        .arg("--untagged")
+        .arg("-b")
+        .arg("--algorithm=md5")
+        .arg(at.subdir.join("f"))
+        .succeeds()
+        .stdout_contains("d41d8cd98f00b204e9800998ecf8427e *");
```

Comment:
Maybe also a test case where `-b` does *not* generate an asterisk, because the file is plain-text.